src/attention/utils/helpers.ts

```diff
@@ -1,98 +1,243 @@
-import { computePosition, flip, offset, shift, arrow, ReferenceElement } from "@floating-ui/dom";
+import {
+  computePosition,
+  flip,
+  offset,
+  shift,
+  arrow,
+  autoUpdate,
+  ReferenceElement,
+} from '@floating-ui/dom'
 
-const TOP = "top";
-const BOTTOM = "bottom";
-const LEFT = "left";
-const RIGHT = "right";
-type Directions = "top" | "right" | "bottom" | "left";
+const TOPSTART = 'top-start'
```

Comment:
It's a small thing, but maybe we could add underscore as separator, like `TOP_START`? I think it would be easier to read.

Reply:
Updated in commit: [6c52c9d](https://github.com/warp-ds/core/pull/15/commits/6c52c9dd87f76c84b60c6c03389dbcd4302f36c2)